modules/appeals_api/spec/requests/v2/supplemental_claims_controller_spec.rb

```diff
@@ -85,6 +85,37 @@ def base_path(path)
       end
     end
 
+    context 'returns 422 when birth date is not a date' do
+      let(:error_content) do
+        { 'status' => 422, 'detail' => "'apricot' did not match the defined pattern" }
+      end
+
+      it 'when given a string for the birth date ' do
+        headers.merge!({ 'X-VA-Birth-Date' => 'apricot' })
+
+        post(path, params: data.to_json, headers: headers)
+        expect(response.status).to eq(422)
+        expect(parsed['errors']).to be_an Array
+      end
+    end
+
+    context 'returns 422 when decison date is not a date' do
+      let(:error_content) do
+        { 'status' => 422, 'detail' => "'banana' did not match the defined pattern" }
+      end
+
+      it 'when given a string for the contestable issues decision date ' do
+        sc_data = JSON.parse(data)
+        sc_data['included'][0]['attributes'].merge!('decisionDate' => 'banana')
+
+        post(path, params: sc_data.to_json, headers: headers)
+        expect(response.status).to eq(422)
+        expect(parsed['errors']).to be_an Array
+        expect(parsed['errors'][0]['title']).to include('Invalid')
+        expect(parsed['errors'][0]['detail']).to include("'banana' did not fit within the defined length limits")
```

Comment:
Same here...is this really what the error returns when submitting something other than a proper date? We'll need to fix this if that's the case!